onadata/apps/api/tests/viewsets/test_abstract_viewset.py

```diff
@@ -106,6 +106,12 @@ def user_profile_data(self):
             'metadata': {},
         }
 
+    def validate_openrosa_head_response(self, response):
+        self.assertEqual(response.status_code, 204)
```

Comment:
this is the convention in these tests; simple integers are used instead of constants from `rest_framework.status`